processor/src/tasks/processReadme.js

```diff
@@ -0,0 +1,127 @@
+const globule = require('globule')
+const path = require('path')
+const superagent = require('superagent')
+const cheerio = require('cheerio')
+
+const { exists, readFile, writeFile } = require('../utils')
+const { GITEA_URL } = require('../env')
+
+function processReadme(events, inputDir, kitspaceYaml, outputDir, projectFullname) {
+  if (kitspaceYaml.multi) {
+    const projectNames = Object.keys(kitspaceYaml.multi)
+    return Promise.all(
+      projectNames.map(projectName => {
+        const projectOutputDir = path.join(outputDir, projectName)
+        const projectKitspaceYaml = kitspaceYaml.multi[projectName]
+
+        return _processReadme(
+          events,
+          inputDir,
+          projectKitspaceYaml,
+          projectOutputDir,
+          projectFullname,
+        )
+      }),
+    )
+  }
+
+  return _processReadme(events, inputDir, kitspaceYaml, outputDir, projectFullname)
+}
+
+async function _processReadme(
+  events,
+  inputDir,
+  kitspaceYaml,
+  outputDir,
+  projectFullname,
+) {
+  const readmePath = path.join(outputDir, 'readme.json')
```

Comment:
Why `.json`?

Reply:
The file extension doesn't really matter. If it were `.html` the only difference in the frontend `res.text()` would be used instead of `res.json()`

Reply:
Except it just obfuscates the actual content and wraps in JSON for no reason. Please make this `.html`.

Reply:
Resolved in a2a5dc0.